packages/api-client/index.ts

```diff
@@ -163,7 +165,17 @@ class SearchAPIClient {
     const coursesData = await res.data.data;
 
     if (coursesData.bulkClasses) {
-      return coursesData.bulkClasses;
+      return coursesData.bulkClasses.map((course) => {
+        if (course === null) return null;
+
+        return {
+          name: course.name,
+          classId: course.classId,
+          subject: course.subject,
+          numCreditsMin: course?.latestOccurrence?.minCredits,
```

Comment:
I had to unwrap the data here bc minCredits is nested within latestOccurence (and we have a different name for it).

Reply:
awesome, we do similar logic in the fetchCourse function... could you abstract it out? We use fetchCourse in getting course coreqs and things I think(may be time to move that to bulk as well but that's for a later task)

Reply:
will do!